server/tm_cmds.go

```diff
@@ -114,7 +114,7 @@ against which this app has been compiled.
 				BlockProtocol uint64
 				P2PProtocol   uint64
 			}{
-				Tendermint:    tversion.Version,
+				Tendermint:    tversion.TMCoreSemVer,
```

Comment:
fyi @tessr we had to change this to be compatible with v0.34.0-rc6. Might be a useful addition to the changelong 🙏

Reply:
Ah, thanks for the heads up!